askama/Cargo.toml

```diff
@@ -31,6 +31,7 @@ with-mendes = ["askama_derive/with-mendes"]
 with-rocket = ["askama_derive/with-rocket"]
 with-tide = ["askama_derive/with-tide"]
 with-warp = ["askama_derive/with-warp"]
+localization = ["dep:fluent-templates","dep:unic-langid"]
```

Comment:
"dep:" is not needed here.

Reply:
Not? Well I guess I miss understood the docs^^ Will implement your suggested changes in the next few hours or so, but thank you for the input so far :)